cmd/cluster.go

```diff
@@ -965,6 +973,9 @@ func (c *config) updateCluster(args []string) error {
 	if viper.IsSet("logacceptedconns") {
 		clusterFeatures.LogAcceptedConnections = &logAcceptedConnections
 	}
+	if viper.IsSet("high-availability-control-plane") {
+		clusterFeatures.HighAvailability = &highAvailability
```

Comment:
If this is toggled from false to true, can we also add a prompt like this?

```golang
	if err := genericcli.PromptCustom(&genericcli.PromptConfig{
		Message:         "Enabling the HA control plane feature gate is still a beta feature. Please be aware that you cannot toggle back this feature after it was enabled.",
		ShowAnswers: true,
		Out:             c.out,
	}); err != nil {
		return err
	}
```

I would like to point out that this feature is still `beta`. So we have a bit more time to collect experience in production with it before motivating all customers to enable it.